server/controllers/index.js

```diff
@@ -77,7 +76,7 @@ const controllers = {
       pgdb
         .addReview(req.body)
         .then((data) => {
-          res.status(200).send(data);
+          res.status(201).send('success');
         })
```

Comment:
Good response code change!  If you want, you can read the error code in your catch block and if it comes back matching a duplicate error, send a 409 for conflict (else send a 500).